[2022-01-11 19:03:30,069] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A scheduled__2022-01-10T00:00:00+00:00 [queued]>
[2022-01-11 19:03:30,084] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A scheduled__2022-01-10T00:00:00+00:00 [queued]>
[2022-01-11 19:03:30,085] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 19:03:30,086] {taskinstance.py:1239} INFO - Starting attempt 3 of 3
[2022-01-11 19:03:30,086] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 19:03:30,192] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): training_model_A> on 2022-01-10 00:00:00+00:00
[2022-01-11 19:03:30,196] {standard_task_runner.py:52} INFO - Started process 5035 to run task
[2022-01-11 19:03:30,204] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_A', 'scheduled__2022-01-10T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpwmz7n9vp', '--error-file', '/tmp/tmppkdtj6ui']
[2022-01-11 19:03:30,222] {standard_task_runner.py:77} INFO - Job 24: Subtask training_model_A
[2022-01-11 19:03:30,353] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.training_model_A scheduled__2022-01-10T00:00:00+00:00 [running]> on host 40c4c8725776
[2022-01-11 19:03:30,569] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_A
AIRFLOW_CTX_EXECUTION_DATE=2022-01-10T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-10T00:00:00+00:00
[2022-01-11 19:03:30,569] {python.py:175} INFO - Done. Returned value was: 3
[2022-01-11 19:03:30,665] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_A, execution_date=20220110T000000, start_date=20220111T190330, end_date=20220111T190330
[2022-01-11 19:03:30,769] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-11 19:03:30,875] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
